docs/source/guides/user/chapters/logging.rst

```diff
@@ -39,9 +40,13 @@ directory by running::
 
   $ avocado run --store-logging-stream STREAM[:LEVEL][,STREAM[:LEVEL][,...]
  
-This will produce ``$STREAM.$LEVEL`` files per each (unique) entry in the test
-results directory.
+This will produce ``$STREAM.$LEVEL`` files per each (unique) entry in two places.
```

Comment:
Could we provide the example command line for this separation? I understand the above described the general approach but how about an example below this paragraph for the case of gathering streams in the job log? So if e.g. I run in serial I would have to do something like `avocado run --store-logging-stream job,test`?

Reply:
sure, I will do that.